<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayerWidget</class>
 <widget class="QWidget" name="VideoPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="showVideo">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>846</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 191, 142);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>380</y>
     <width>838</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>92</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>已经播放</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>400</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>24</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rationcomboBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>0.5X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.0X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.0X</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>总时长:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
